tests/config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I managed to figure out how to sanely move both the log and the configuration file to the appropriate temporary directory for the OS.  You can see both changes in the latest commit.  I hope you will be pleased to see that the fix makes all configuration options addressable through the options block of the component.